android/build.gradle

```diff
@@ -36,6 +36,7 @@ android {
     dependencies {
         implementation 'com.segment.analytics.android:analytics:4.10.0'
         implementation 'com.segment.analytics.android.integrations:amplitude:3.0.3'
+        implementation 'com.appsflyer:segment-android-integration:+'
```

Comment:
Please declare the version, it is not a good idea to have open versions because this may break in the future because of a release from them